.gitignore

```diff
@@ -97,3 +97,4 @@ Andika-R.ttf
 Bloom/
 .vs/config/applicationhost.config
 build/Bloom*.nupkg
+src/BloomBrowserUI/filesToFix.lst
```

Comment:
Building on Linux from the command line. Gets created by `src/BloomBrowserUI/fixNodejsForLinux.sh`.